app/overrides/controllers/decidim/consultations/question_votes_controller.rb

```diff
@@ -17,21 +17,15 @@ def delegation
     @delegation ||= Decidim::ActionDelegator::Delegation.find_by(id: params[:decidim_consultations_delegation_id])
   end
 
-  def enforce_permission_to_unvote(subject, extra_context = {})
+  def enforce_permission_to_unvote
     if delegation.blank?
       enforce_permission_to :unvote, :question, question: current_question
     else
-      if Rails.env.development?
-        Rails.logger.debug "==========="
-        Rails.logger.debug [permission_scope, :unvote, subject, permission_class_chain].map(&:inspect).join("\n")
-        Rails.logger.debug "==========="
-      end
-
       raise Decidim::ActionForbidden unless allowed_to?(
-        :unvote,
-        subject,
-        extra_context,
-        [Decidim::Consultations::Permissions, Decidim::Admin::Permissions, Decidim::Permissions],
+        :unvote_delegation,
+        :question,
+        { question: current_question, delegation: delegation },
+        [Decidim::ActionDelegator::Permissions, Decidim::Admin::Permissions, Decidim::Permissions],
```

Comment:
This specific chain, used in other places, is a clear indication that the caller needs a new controller of our own so that the permission chain can be properly specified from there.